### ISSUE TITLE
Test case "BatchNorm1d_3d_input_eval" is wrong

### ISSUE THREAD

**Issue by snnn** [onnx/onnx#732]
Test case "BatchNorm1d_3d_input_eval" is converted from pytorch. This is designed for PyTorch's  BatchNorm1d OP, which take a 2D or 3D tensor as input.
However, BatchNorm OP in onnx only accepts 4-D input.

**Comment by houseroad** [comment]
We will fix the spec here: https://github.com/onnx/onnx/pull/733/files

**Comment by snnn** [comment]
Close as #733 is merged.
